src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -338,6 +340,55 @@ class Service : public vsdk::MLModelService, public vsdk::Stoppable, public vsdk
         return state_;
     }
 
+    static std::string initialize_directory_(std::string model_name) {
```

Comment:
Remember to add these to the .hpp file, too! It's not strictly necessary for private functions like this, but it's good practice.

Reply:
No, the .hpp file doesn't contain anything! The entire class is in the .cpp file, without adding headers anywhere. :shrug:

Reply:
There's no rule that says you need a header file in C++. This class is entirely local, nothing consumes it except via the abstract interfaces it implements. So there is no header required to describe the type to other consumers.